### DISCUSSION TITLE
In version 0.9.2, assuming that the table field name data type is VARCHAR [], Python reads it using dataframe. to_dict (orientation='records'), and the data is of type list. However, when upgrading to 1.0.0, the data becomes of type numpy.nparay. Is this correct?

### DISCUSSION THREAD

**Discussion by bitszhang** [duckdb/duckdb#12530] [General]
In version 0.9.2, assuming that the table field name data type is VARCHAR [], Python reads it using dataframe. to_dict (orientation='records'), and the data is of type list. However, when upgrading to 1.0.0, the data becomes of type numpy.nparay. Is this correct?

**Comment by Tishj** [comment]
Yes
https://github.com/duckdb/duckdb/pull/10826
